cpp/src/parquet/CMakeLists.txt

```diff
@@ -168,6 +167,7 @@ set(PARQUET_SRCS
     murmur3.cc
     "${ARROW_SOURCE_DIR}/src/generated/parquet_constants.cpp"
     "${ARROW_SOURCE_DIR}/src/generated/parquet_types.cpp"
+    "${ARROW_SOURCE_DIR}/src/arrow/util/qpl_job_pool.cc"
```

Comment:
Why do we need to add this here not `cpp/src/arrow/`?

Reply:
no need for this